scripts/bungie-data-helper.coffee

```diff
@@ -86,9 +122,13 @@ class DataHelper
       if node.isActivated
         name = "*#{step.nodeStepName}*"
       text[column] = "" unless text[column]
-      text[column]+= name + " | "
+      text[column] += (if text[column] then ' | ' else '') + name
 
     setText node for node in nodes
-    return text
+    stats = []
+    for statName, statValue of item.stats
+        stats.push "#{statName} : #{statValue}"
```

Comment:
nitpicky, but could we get rid of the space before the ":" just to condense things a bit more?